server/app/src/main/java/com/google/heartrate/wearos/app/MainActivity.java

```diff
@@ -18,66 +25,76 @@
  * <p>Used only for testing {@link BluetoothServer} with {@link HeartRateGattService} hosted.
```

Comment:
Are you sure you mean testing? This is not testing code used in java tests, is it?